src/components/Table/__tests__/Table.withTableSettings.test.tsx

```diff
@@ -288,7 +288,7 @@ describe('withTableSettings', () => {
 
             await userEvent.click(screen.getByRole('button', {name: 'Table settings'}));
             await userEvent.click(await screen.findByRole('button', {name: 'description'}));
-            await userEvent.click(screen.getByRole('button', {name: 'Apply'}));
+            await userEvent.click(screen.getByRole('button', {name: 'button_apply'}));
```

Comment:
have no idea why it is broken now

Reply:
This is because you test component `const TableWithSettings = withTableSettings<SomeItem>(Table);` . And keysets are taken from https://github.com/gravity-ui/uikit/tree/839d503f378e23eef87887866d20973bcd174fb4/src/components/Table/hoc/withTableSettings/i18n

Reply:
In separate PR, I do not know what happened with this place